auto_install/install.sh

```diff
@@ -57,8 +57,8 @@ r=$(( r < 20 ? 20 : r ))
 c=$(( c < 70 ? 70 : c ))
 
 # Find IP used to route to outside world
-IPv4addr=$(ip route get 8.8.8.8| awk '{print $7}')
-IPv4gw=$(ip route get 8.8.8.8 | awk '{print $3}')
+IPv4addr=$(ip route get 9.9.9.9 | awk '{print $7}')
+IPv4gw=$(ip route get 9.9.9.9 | awk '{print $3}')
```

Comment:
This can be any public IP address since it doesn't actually contact the server. While we are here can you replace the references with 192.0.2.1?

From [RFC 5735](https://tools.ietf.org/html/rfc5735):
> 192.0.2.0/24 - This block is assigned as "TEST-NET-1" for use in documentation and example code.  It is often used in conjunction with domain names example.com or example.net in vendor and protocol documentation.  As described in [RFC5737], addresses within this block do not legitimately appear on the public Internet and can be used without any coordination with IANA or an Internet registry.  See [RFC1166].